parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -4,16 +4,19 @@ Port 22
 #ListenAddress ::
 #ListenAddress 0.0.0.0
 Protocol 2
+
+# 5.2.11 Ensure only approved MAC algorithms are used
+MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+
+# 5.2.12 Ensure SSH Idle Timeout Interval is configured
+ClientAliveInterval 300
+ClientAliveCountMax 3
+
 # HostKeys for protocol version 2
 HostKey /etc/ssh/ssh_host_rsa_key
 HostKey /etc/ssh/ssh_host_dsa_key
 HostKey /etc/ssh/ssh_host_ecdsa_key
 HostKey /etc/ssh/ssh_host_ed25519_key
-
-KexAlgorithms curve25519-sha256@libssh.org
```

Comment:
We should probably also restore this, as removing it adds less secure algorithms.

Reply:
added back above